docs/source/guides/develop-iroha-module.md

```diff
@@ -125,10 +125,10 @@ And see what it does to register a new Bridge:
 
 We will not discuss Bridge-related terminology here – the thing we want to look at is how we can compose these steps into one new Iroha Special Instruction.
 
-As you can see, we have `Instruction::If(...)` here. It's [the utility Iroha Special Instruction](references/glossary#utility-iroha-special-instruction).
+As you can see, we have `Instruction::from(...)` here. It's [the utility Iroha Special Instruction](references/glossary#utility-iroha-special-instruction).
```

Comment:
This paragraph doesn't make sense with this change

Reply:
a lot of things in this file are outdated

Reply:
Remove the file. It's better to have no docs than misleading ones.